products/bigquerydatatransfer/api.yaml

```diff
@@ -138,6 +138,18 @@ objects:
               - schedule_options.0.disable_auto_scheduling
               - schedule_options.0.start_time
               - schedule_options.0.end_time
+      - !ruby/object:Api::Type::NestedObject
+        name: 'emailPreferences'
+        description: |
+          Email notifications will be sent according to these preferences to the
+          email address of the user who owns this transfer config.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableFailureEmail'
+            description: |
+              If true, email notifications will be sent on transfer run failures.
+            at_least_one_of:
+              - email_preferences.0.enable_failure_email
```

Comment:
I think it makes more sense to use `required: true` instead of `at_least_one_of` here.

Reply:
Fair point! https://github.com/GoogleCloudPlatform/magic-modules/pull/4156/commits/fc9f7d0835ce51d53abb505d59553857c421aa92

Reply:
Thanks @samouss! Looks good to me, I'll get this merged